src/platform/packages/shared/kbn-scout/src/playwright/fixtures/test/browser_auth/index.ts

```diff
@@ -64,6 +78,16 @@ export const browserAuthFixture = coreWorkerFixtures.extend<{ browserAuth: Brows
     };
 
     log.serviceLoaded('browserAuth');
-    await use({ loginAsAdmin, loginAsViewer, loginAsPrivilegedUser });
+    await use({
+      loginAsAdmin,
+      loginAsViewer,
+      loginAsPrivilegedUser,
+      loginWithCustomRole,
+    });
+
+    if (isCustomRoleCreated) {
+      log.debug('Deleting custom role');
```

Comment:
(Absolutely optional): should we log the custom role name we're about to delete?

```suggestion
      log.debug(`Deleting custom role with name ${samlAuth.customRoleName}`);
```

I propose this because it may help developers understand exactly which role we're about to delete (no guessing).